share/spack/gitlab/cloud_pipelines/stacks/e4s-oneapi/spack.yaml

```diff
@@ -38,7 +40,9 @@ spack:
       operating_system: ubuntu20.04
       target: x86_64
       modules: [compiler]
-      environment: {}
+      environment:
+        prepend_path:
+          LD_LIBRARY_PATH: /opt/intel/oneapi/compiler/2022.1.0/linux/compiler/lib/intel64_lin
```

Comment:
Is this affecting the binaries that get generated in any way? For instance, can people relocate them is they end up in a public buildcache?

Reply:
It should not affect the generated binaries. It is needed because the compiler needs to find its own shared libraries during compile-time. The generated binaries will use the runtime value of the LD_LIBRARY_PATH to find the libraries.

Reply:
That is fine, I was wondering if any of the library in there will get linked into an artifact and not RPATHed.